### PR TITLE
Fix GTest Version

### PR DESCRIPTION
**Pull request by sslattery**
revert to stable gtest version

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -4,22 +4,22 @@ cmake_minimum_required(VERSION 3.12)
...
-configure_file(CMakeLists.txt.in googletest-download/CMakeLists.txt)
-execute_process(COMMAND "${CMAKE_COMMAND}" -G "${CMAKE_GENERATOR}" .
-    WORKING_DIRECTORY "${CMAKE_BINARY_DIR}/googletest-download" )
-execute_process(COMMAND "${CMAKE_COMMAND}" --build .
-    WORKING_DIRECTORY "${CMAKE_BINARY_DIR}/googletest-download" )
+include(FetchContent)
+FetchContent_Declare(
+  googletest
+  GIT_REPOSITORY https://github.com/google/googletest.git
+  GIT_TAG        release-1.10.0
```
Comment (dalg24) [inline CMakeLists.txt]: Any reason to prefer this to the tarball?

Comment (sslattery) [reply CMakeLists.txt]: We had been using `master` which proved to be less stable than advertised. This is the latest stable release.

Comment (sslattery) [reply CMakeLists.txt]: Fixed - now using URL and tarball instead

**Comment by sslattery** [comment]
@junghans Any opinion on this one?

**Pull request merged**